R/database.R

```diff
@@ -2628,3 +2630,385 @@ get_gene_expression = function(metadata,
   }
   return(expression_wider)
 }
+
+
+#' Load the manta output for a set of samples
+#'
+#' This is a convenience wrapper function for get_manta_sv_by_sample. See that function for more information.
+#'
+#' @param these_samples_metadata The only required parameter is a metadata table (data frame) that must contain a row for each sample you want the data from. The additional columns the data frame needs to contain, besides sample_id, are: unix_group, genome_build, seq_type, pairing_status
+#' @param min_vaf The minimum tumour VAF for a SV to be returned.
+#' @param min_score The lowest Manta somatic score for a SV to be returned.
+#' @param pass If set to TRUE, include SVs that are annotated with PASS in FILTER column. Default is TRUE.
+#' @param with_chr_prefix Prepend all chromosome names with chr (required by some downstream analyses).
+#' @param from_flatfile Set to TRUE by default.
+#' @param projection The projection genome build.
```

Comment:
I am having a hard time following the whole all_sv_hg83/all_sv_grch37 setup. Seems like you're making the user do the work and supply what's available. Why doesn't the function work this out for the user?


There's also a typo in your documentation (83 instead of 38).

Reply:
I am guessing you did this so this function doesn't need to call get_manta_sv multiple times. Is that right?

Reply:
Yes, That is why I added this as a parameter. For this function, I have now removed these parameters. The function now loads all manta sv calls, based on the selected projection.